packages/flutter_driver/lib/src/common/diagnostics_tree.dart

```diff
@@ -69,7 +69,7 @@ class DiagnosticsTreeResult extends Result {
 
   /// The JSON encoded [DiagnosticsNode] tree requested by the
   /// [GetDiagnosticsTree] command.
-  final Map<String, Object> json;
+  final Map<String, dynamic> json;
```

Comment:
Not `Object?`?

Reply:
The return type of the method on result is defined as:

```
 /// Serializes this message to a JSON map.
  Map<String, dynamic> toJson();
```

Reply:
FWIW, I would lean towards moving from `dynamic` to `Object?` everywhere rather than the other way around, but that's something we can worry about in another PR.